src/ralph/networks/api.py

```diff
@@ -94,14 +101,19 @@ def destroy(self, request, *args, **kwargs):
 
 class NetworkViewSet(RalphAPIViewSet):
     queryset = Network.objects.all()
-    serializer_class = NetworkSerializer
     select_related = ['network_environment', 'kind']
     prefetch_related = ['racks']
     extended_filter_fields = {
         # workaround for custom field for address field defined in admin
         'address': ['address'],
     }
 
+    def get_serializer_class(self):
```

Comment:
use `serializer_class` and `save_serializer_class` class attributes instead

Reply:
Fixed